File: app/src/main/java/com/locallinkonline/stcatherineschool/rest/model/LunchSchedule.java
package com.locallinkonline.stcatherineschool.rest.model;

import com.google.gson.annotations.SerializedName;
import com.locallinkonline.stcatherineschool.room.entity.LunchEntity;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;

@EqualsAndHashCode
@NoArgsConstructor
@Getter
public class LunchSchedule {
    @SerializedName("alexaResponse")
    String alexaResponse;
    @SerializedName("lunchScheduleArray")
    LunchEntity[] lunches;
}

File: app/src/main/java/com/locallinkonline/stcatherineschool/rest/model/LunchSchedule.kt
package com.locallinkonline.stcatherineschool.rest.model

import com.google.gson.annotations.SerializedName
import com.locallinkonline.stcatherineschool.room.entity.LunchEntity

data class LunchSchedule(
        @SerializedName("alexaResponse")
        var alexaResponse: String?,
        @SerializedName("lunchScheduleArray")
        var lunches: List<LunchEntity>? = null
)
